crates/store/re_video/Cargo.toml

```diff
@@ -31,9 +31,11 @@ av1 = ["dep:dav1d"]
 
 ## Enable faster native video decoding with assembly.
 ## You need to install [nasm](https://nasm.us/) to compile with this feature.
-# TODO(#7671): this feature flag currently does nothing
-# nasm = ["dav1d?/default"] # The default feature set of dav1d has asm enabled
-nasm = [] ## TODO(#7671): fix Linux build
+# TODO(#7671): this feature flag currently does nothing on Linux.
+nasm = [
+  # The default feature set of our dav1d fork has asm for linux enabled.
+  "dav1d?/default",
```

Comment:
These comments seem to contradict each other.

Did you mean this?

```suggestion
# TODO(#7671): this feature flag currently does nothing on Linux.
nasm = [
  # The default feature set of our dav1d fork has asm enabled (except on Linux, see above)
  "dav1d?/default",
```

Reply:
good catch, thx